src/BackwardsCompatiblePayloadSerializerFactory.php

```diff
@@ -420,6 +418,9 @@ function (array $serializedObject) use ($contactPointUpdatedEvent) {
             );
         }
 
+        self::fillDescriptions($payloadManipulatingSerializer);
```

Comment:
If you follow the existing approach then the code will look like

```
$updateDescriptionEvents = [
  EventDescriptionUpdated::class,
  PlaceDescriptionUpdated::class,
];
foreach ($updateDescriptionEvents as $updateDescriptionEvents) {
   ...
   self::fillDescriptions(serializedObject)
   self::replacePlaceIdWithItemId($serializedObject)
   ...
}
```

The reasoning behind this was that:
- This makes it more clear which events are manipulated
- The update methods are very granular and are building blocks